.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Use unix line endings everywhere, even on Windows
+* text=auto eol=lf
```

Comment:
does this disallow people from the git setting "check out with windows/commit with Unix"

Reply:
No, I don't believe it does. Git should continue to do its magic for devs working on Windows, as far as I can tell.

Reply:
Let me know if you find otherwise. I have been playing around with the `git config --global core.autocrlf true` settings locally, so my experience and dev box might be in a non-default state.